spec/cedar_policy/authorizer_spec.rb

```
@@ -172,5 +172,152 @@
+
+        it { is_expected.to(have_attributes(decision: CedarPolicy::Decision::ALLOW)) }
+      end
+
+      describe "Non-Owner cannot Edit image" do
+        let(:principal) { CedarPolicy::EntityUid.new("User", "3") }
+        let(:action) { CedarPolicy::EntityUid.new("Action", "Edit") }
+        let(:entities_data) {
+          [
+            {uid: {type: "User", id: "1"}, attrs: {}, parents: []},
+            {uid: {type: "User", id: "3"}, attrs: {}, parents: []},
+            {uid: {type: "Image", id: "1"}, attrs: {owner: {type: "User", id: " 1"}}, parents: []}
```

rubocop (1.68.0)

[convention] Space inside { missing.

(Layout/SpaceInsideHashLiteralBraces)